lib/LaTeXML/Post/OpenMath.pm

```diff
@@ -160,7 +160,8 @@ sub om_expr_aux {
   elsif ($tag eq 'ltx:XMText') {
     if (scalar(element_nodes($node))) {    # If it has markup?
       return ['om:OMATTR', {},
-        ['om:OMATP', { cd => 'OMDoc', name => 'verbalizes' },
+        ['om:OMATP', {},
+        ['om:OMS', { cd => 'OMDoc', name => 'verbalizes' }],
```

Comment:
This change on the other hand is something I added, and is sTeX-specific, so it is quite easy to motivate changing it.